packages/@o3r-training/showcase-sdk/project.json

```diff
@@ -36,6 +36,21 @@
       "options": {
         "command": "npm publish packages/@o3r-training/showcase-sdk/dist"
       }
+    },
+    "extract-folder-structure": {
+      "cache": true,
+      "executor": "nx:run-script",
+      "options": {
+        "script": "extract"
+      },
+      "inputs": [
```

Comment:
The following file seems missing `{projectRoot}/openapitools.json`
(if not already part of `source`)